### PR TITLE
Add tensorflow to gitlab ci

### PR DESCRIPTION
**Pull request by FHoffmannCode**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -246,6 +256,48 @@ dbnd_airflow 0 1_10_9_py36:
...
+    IMAGE: python:3.6
+    TEST_NAME: py36-tensorflow1
+
+dbnd_tensorflow 2 py36:
+  <<: *template__test_tensorflow
+  variables:
+    IMAGE: python:3.6
+    TEST_NAME: py36-tensorflow2
+
+dbnd_tensorflow 1 py37:
```
Comment (evgenyshulman) [inline .gitlab-ci.yml:289]: let's remove it from the ci/cd and keep only tensorflow 1 + py27/py36,   tensorflow 2 + py36

(all others are still available in tox)

Comment (FHoffmannCode) [reply .gitlab-ci.yml:289]: okay. whats the reason for that?

Comment (evgenyshulman) [reply .gitlab-ci.yml:289]: minimize Ci/CD jobs we need to run. ( right now I think it will cover 99% of the problems)

Comment (evgenyshulman) [reply .gitlab-ci.yml:289]: we can just mark them as manual steps for now

Comment (FHoffmannCode) [reply .gitlab-ci.yml:291]: good idea, done

**Review by turbaszek — APPROVED** [review]

**Review by evgenyshulman — APPROVED** [review]

**Pull request merged**